client/react/frontpage/components/SocialPanel.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+
+import './SocialPanel.scss';
+
+// update social media links here
+// img is the base name of the Font Awesome icon
+const socialMediaLinks = [
+  {
+    img: 'facebook',
+    link: 'https://www.facebook.com/uclaradio',
+  },
+  {
+    img: 'twitter',
+    link: 'https://twitter.com/UCLAradio',
+  },
+  {
+    img: 'instagram',
+    link: 'https://www.instagram.com/uclaradio/',
+  },
+  {
+    img: 'snapchat-ghost',
+    link: 'https://www.snapchat.com/add/uclaradio',
+  },
+];
+
+const SocialPanel = React.createClass({
+  convertToFaName(img, size) {
+    return 'fa fa-' + img + ' fa-' + size;
+  },
+  render() {
+    const self = this;
+    return (
+      <div>
+        <p className="infoHeader">Social Media</p>
+        <div className="link-container">
+          {socialMediaLinks.map(function(obj) {
+            // change size here
+            let faName = self.convertToFaName(obj.img, '2x');
+            return (
+              <a href={obj.link} className="social-link">
```

Comment:
You should check out the code in [ShowPage.js](https://github.com/uclaradio/uclaradio/blob/master/client/react/frontpage/components/ShowPage.js), [ShowPage.scss](https://github.com/uclaradio/uclaradio/blob/master/client/react/frontpage/components/ShowPage.scss), and [_common.scss](https://github.com/uclaradio/uclaradio/blob/master/client/react/frontpage/_common.scss). I implemented a similar feature for shows, and I think you could steal some styles from that.

Also, I really like how you implemented this with `.map`, so if you want to refactor ShowPage.js, that'd be 💯

Reply:
+1 to your use of map to produce this social media panel. I'd love to see your Social Media panel utilized in our ShowPage component.